quic/s2n-quic-core/src/connection/error.rs

```diff
@@ -156,6 +156,71 @@ impl fmt::Display for Error {
     }
 }
 
+impl PartialEq for Error {
+    #[inline]
+    fn eq(&self, other: &Self) -> bool {
+        // ignore the `source` attribute when considering if errors are equal
```

Comment:
what is making the test fail only for rust beta?

Reply:
for some reason the beta version started calculating sources differently and we're now getting failures from it. i didn't dig in too much, since it's a good idea to avoid comparing line numbers anyway as part of equivalence.